SQL/migrate/V060__headset_styles.sql

```diff
@@ -0,0 +1,7 @@
+--
+-- Implemented in PR #10540.
+-- Adds a `headset_choice` column for headset type preferences.
+--
+
+ALTER TABLE `ss13_characters`
+	ADD COLUMN `headset_choice` TINYINT(11) NULL DEFAULT NULL AFTER `pda_choice`;
```

Comment:
Tinyint display width should be smaller. Idk if we even need to specify width since you're not doing `ZEROFILL` ?